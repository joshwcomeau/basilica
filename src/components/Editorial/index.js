// eslint-disable-next-line no-unused-vars
import React, { Component, PropTypes } from 'react';
import classNames from 'classnames';

import './index.scss';


const Editorial = () => {
  const classes = classNames(['editorial']);

  return (
    <div className={classes}>
      Words and Stuff
    </div>
  );
};

Editorial.propTypes = {

};

Editorial.defaultProps = {

};

export default Editorial;
