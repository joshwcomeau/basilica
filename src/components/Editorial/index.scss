@import '../../variables';

.editorial {
  min-height: 1200px;
  background: #FFF;

  @media screen and (max-width: $mq-tablet) {

  }

  @media screen and (max-width: $mq-mobile) {

  }
}
