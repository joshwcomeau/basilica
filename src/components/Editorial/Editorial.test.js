/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Editorial from './index';

describe('Editorial', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Editorial />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
