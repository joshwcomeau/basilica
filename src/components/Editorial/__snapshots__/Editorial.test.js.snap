exports[`Editorial renders correctly 1`] = `
<div
  className="editorial">
  Words and Stuff
</div>
`;
